### PR TITLE
Maven + Tycho build system support

### PR DESCRIPTION
**Pull request by EvgeniyaKatunina**
This PR is for the future release and the commits 6c57796, e9e9db5 reverted with the 8ddb5c1, b8d4052 commits accordingly must be retained before merging this branch to provide compatibility with the CDT 9.0 used by the Eclipse Neon.

### CODE REVIEW

**Comment by anthony-kolesov** [comment]
Dmitrii to test that resulting plugin works on a new Eclipse Neon.

File: pom.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
 	</repository>
 	<repository>
 		<id>eclipse-neon</id>
-		<url>http://ftp-stud.fht-esslingen.de/pub/Mirrors/eclipse/releases/neon/</url>
+		<url>file:/remote/us01home51/arcgnu_verif/prerequisites/eclipse/eclipse_2017.09_p2_repository/</url>
```
Comment (DmitriiKolesnichenko) [inline pom.xml]: Are you sure that this path to our local directory has to be hard coded?

Comment (DmitriiKolesnichenko) [inline pom.xml]: Are you sure that this path to our local directory has to be hard coded?

Comment (EvgeniyaKatunina) [inline pom.xml]: Fixed this in the next comment (7777415).

**Pull request closed without merging**